### DISCUSSION TITLE
如何通过命令控制vue-cesium执行系统时间

### DISCUSSION THREAD

**Discussion by kfcvs** [zouyaoji/vue-cesium#178] [Q&A] [answered]
想利用vue-cesium框架进行二次开发，但通过原生 CesiumJS API 中未找到执行系统时间的相关命令，请大佬们帮忙解答

**Comment by zouyaoji** [comment, marked as answer]
请描述具体一点呢，Cesium提供了https://cesium.com/learn/cesiumjs/ref-doc/JulianDate.html类来处理哈。
